libcloudforensics/providers/gcp/internal/bigquery.py

```diff
@@ -0,0 +1,59 @@
+# -*- coding: utf-8 -*-
+# Copyright 2021 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Google BigQuery functionalities."""
+
+from typing import TYPE_CHECKING, List, Dict, Any, Optional
+from libcloudforensics.providers.gcp.internal import common
+
+if TYPE_CHECKING:
+  import googleapiclient.discovery
+
+
+class GoogleBigQuery:
+  """Class to call Google BigQuery APIs.
+
+  Attributes:
+    project_id: Google Cloud project ID.
+  """
+  BIGQUERY_API_VERSION = 'v2'
```

Comment:
I know you followed the convention in other files, but here it makes more sense to make this a module variable rather than a class variable, you can then use it right away by name rather than via self.BIG...
Also if this will be only called in this module, you should prefix it with" _"